### PR TITLE
Feature 1 - Display Train List

### PR DESCRIPTION
**Pull request by Mani375**
###Train Details Feature Added

### CODE REVIEW

File: src/main/java/in/mani/service/TrainDetailSevices.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+public class TrainDetailSevices {
+	
+	private TrainDetailSevices() {
+	//Default Constructor
+	}
+
+	private static Map<Long, String> trainList = new HashMap<>();
+	// Default adding the Train Name and Train Number
+	static {
+		trainList.put(12613L, "TEJAS EXPRESS");
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/mani/service/TrainDetailSevices.java]: Choose appropriate data type - TrainNo . Prefer : Integer

File: src/main/webapp/trainlist.jsp [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+				</tr>
+
+				<%
+				for (String trainClassType : trainClass.keySet()) {
+				%>
+				<tr>
+					<td><%=trainClassType%></td>
+					<td>
+						<%
+						out.println(trainClass.get(trainClassType));
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/trainlist.jsp]: Use JSP expression tag ( e.g: <%=trainClass.get(trainClassType)%>

File: src/main/java/in/mani/service/TrainDetailSevices.java (hunk 1 above)
Comment (Mani375) [reply src/main/java/in/mani/service/TrainDetailSevices.java]: Changed datatype Long to Integer for Train Number

File: src/main/webapp/trainlist.jsp (hunk 2 above)
Comment (Mani375) [reply src/main/webapp/trainlist.jsp]: Changed the JSP Expression Tag.

File: src/main/webapp/trainlist.jsp [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<%@page import="java.util.HashMap"%>
+<%@page import="java.util.Map"%>
+<%@page import="java.util.ArrayList"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/trainlist.jsp]: Remove unused import - ArrayList

File: src/main/webapp/trainlist.jsp [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<%@page import="java.util.HashMap"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/trainlist.jsp]: Remove unused import HashMap

**Review by ghost — COMMENTED** [review]
Added comments. Remove unused imports.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**